schema/CreativeWork.schema.yaml

```
@@ -31,6 +31,12 @@ properties:
           anyOf:
             - $ref: Person
             - $ref: Organization
+  comments:
+    '@id': schema:comment
```

The `@id` on the other schema is capitalized, should this follow suit as well?

```suggestion
    '@id': schema:Comment
```

---

No `schema:comment` refers to the _property_ https://schema.org/comment, `schema:Comment` refers to the type `https://schema.org/Comment